src/bee.ts

```diff
@@ -541,6 +593,22 @@ export class Bee {
     return true
   }
 
+  /**
+   * Getter for default postage batchId
+   */
+  getDefaultPostageBatchId(): Address | undefined {
+    return this.postageBatchId
+  }
+
+  /**
+   * Setter for default postage batchId
+   * @param batchId
+   */
+  setDefaultPostageBatchId(batchId: Address | string): void {
```

Comment:
Is the setter really necessary? I would prefer to keep the `Bee` class readonly.

Reply:
I knew you would object to this and I agree with your thinking, but I think it is necessary. 

First of all, it is necessary to have access to `Bee` class to actually create/retrieve the stamps, so for example for the first time running application you would have to create a temporary `Bee` instance to only create the postage stamp and only then you could create the "real" `Bee` instance with the default stamp set. I would argue this is too cumbersome.  

Also, I think the postage batches are quite dynamic. You might have to switch between them, create a new one, and so on and whenever you would like to change the default batch stamp, then you would have to recreate the `Bee` class which again I think is too cumbersome.

Reply:
```suggestion
  set defaultPostageBatchId(batchId: Address | string): void {
```
Can't we make this a real setter?

I also prefer not to have this. If user wants to have such abstraction, they can do this on their own extending the Bee class